app/controllers/hub/users_controller.rb

```diff
@@ -28,6 +28,18 @@ def update
       end
     end
 
+    def resend_invitation
+      user = User.find_by(id: params[:user_id])
+      ability = Ability.new(current_user)
+
+      if ability.can?(:manage, user)
```

Comment:
`current_ability` is a helper that is added to ApplicationController, and is the same as `Ability.new(current_user)`

Reply:
Neat!